src/autoscaler/api/broker/broker_test.go

```diff
@@ -219,4 +219,64 @@ var _ = Describe("Broker", func() {
 			})
 		})
 	})
-})
+
+	Describe("Bind", func() {
+		var ctx context.Context
+		var instanceID string
+		var bindingID string
+		var details domain.BindDetails
+
+		BeforeEach(func() {
+			ctx = context.Background()
+			instanceID = "some_instance-id"
+			bindingID = "some_binding-id"
+		})
+		Context("Create a binding", func() {
+			It("Fails when the additional config-parameter “app-guid” is provided", func() {
+				var bindingParams = []byte(`
+{
+  "configuration": {
+	"app-guid": "8d0cee08-23ad-4813-a779-ad8118ea0b91",
```

Comment:
Thought: 
The user has to give the app-guid as part of the binding process ? If yes, looks too much data entry for the user. If app_guid is required, we can extract the app_guid internally